doc/reference/tooling/tt_cli/binaries.rst

```diff
@@ -1,23 +1,88 @@
 .. _tt-binaries:
 
-Showing a list of installed binaries
-====================================
+Managing binaries in the current environment
+============================================
 
 ..  code-block:: console
 
-    $ tt binaries
+    $ tt binaries COMMAND [PROGRAM_NAME] [VERSION]
```

Comment:
> tt binaries COMMAND [PROGRAM_NAME] [VERSION]

Given that now all commands accept `[PROGRAM_NAME] [VERSION]`, shouldn't we use smth like `[COMMAND_OPTION ...]` here?